_data/projects/Frapp.yml

```diff
@@ -0,0 +1,12 @@
+name: Frapp
+desc: Video Game Giveaways at the palm of your hands
+site: https://github.com/Brianali-codes/Frapp
+tags:
+  - react-native
+  - development
+  - javascript
+  - react-native-paper
+  - expo
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/Brianali-codes/Frapp/labels/up%20for%20grabs
```

Comment:
There are no open issues in this repository, and this label doesn't exist. Without any tasks to work on this project is not going to be displayed on the site...